server/rest/src/main/java/org/infinispan/rest/resources/CacheResource.java

```diff
@@ -174,7 +173,7 @@ private RestResponse putValueToCache(RestRequest request) {
             boolean useAsync = request.getPerformAsyncHeader();
             Long ttl = request.getTimeToLiveSecondsHeader();
             Long idle = request.getMaxIdleTimeSecondsHeader();
-            return putInCache(responseBuilder, useAsync, cache, key, data, ttl, idle, oldData);
+            return putInCache(responseBuilder, useAsync, cache, key, data, ttl, idle);
```

Comment:
Isn't this breaking the case when the request includes an `ETag`? I would rather keep that working rather than support ill-defined objects.

Reply:
What scenario specifically? We have ETag using PUT tests on ```BaseRestOperationsTest```, but I can extend them